### ISSUE TITLE
entitlements are applied to wrong target

### ISSUE THREAD

**Issue by cronn-leonn** [ionic-team/capacitor-configure#30]
Using the following configuration, the entitlements are applied to the App target, not the App Clip target

```
vars:
  BUNDLE_ID:
    default: com.example.app

platforms:
  ios:
    targets:
      App:
        bundleId: $BUNDLE_ID

      AppClip:
        bundleId: $BUNDLE_ID.Clip

        entitlements:
          - aps-environment: development
          - parent-application-identifiers: [$(AppIdentifierPrefix)$BUNDLE_ID]
```

**Comment by mlynch** [comment]
Thanks will look into this soon

**Comment by mlynch** [comment]
Fixed in 1.0.26
